src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -132,8 +136,10 @@ def update_stats(self):
             namespace = self.port_index_namespace[if_index]
 
             # The first half of queue id is for ucast, and second half is for mcast
-            # To simulate vendor OID, we wrap queues by half distance
-            pq_count = math.ceil((max(if_queues) + 1) / 2)
+            # To simulate vendor OID, we wrap queues by max priority groups
+            port_max_queues = self.statedb.get_all(self.statedb.STATE_DB,
```

Comment:
Will this param be same for all asic namespaces in multi-asic platform?
If it will be same, then this could be read from the first asic namespace.
self.db_conn[1].get_all()
or could use Namespace.dbs_get_all(self.db_conn..) -- this will return a dict with data from all namespaces.